hypha/static_src/src/sass/apply/components/_all-submissions-table.scss

```diff
@@ -15,10 +15,8 @@
 
         th {
             &.reviews_stats { // stylelint-disable-line selector-class-pattern
-                color: $color--black-60;
-
                 div {
-                    font-size: 13px;
+                    font-size: 12px;
```

Comment:
When possible I prefer to use "rem" consistently for font size now that we are overhauling it.

Reply:
Updated.